pkg/asset/machines/master.go

```diff
@@ -752,13 +808,40 @@ func IsMachineManifest(file *asset.File) bool {
 	} else if matched {
 		return true
 	}
+	if matched, err := filepath.Match(masterIpAddressFileNamePattern, filename); err != nil {
+		panic("bad format for master ip address file name pattern")
+	} else if matched {
+		return true
+	}
+	if matched, err := filepath.Match(masterIpClaimFileNamePattern, filename); err != nil {
+		panic("bad format for master ip address claim file name pattern")
+	} else if matched {
+		return true
+	}
 	if matched, err := filepath.Match(workerMachineSetFileNamePattern, filename); err != nil {
 		panic("bad format for worker machine file name pattern")
 	} else {
 		return matched
 	}
 }
 
+// IPAddresses returns IPAddress manifest structures.
+func (m *Master) IPAddresses() ([]ipamv1.IPAddress, error) {
+	var ipAddresses []ipamv1.IPAddress
+	for i, file := range m.IpAddrFiles {
+		logrus.Debugf("Attempting to load address %v.", file.Filename)
+		address := &ipamv1.IPAddress{}
+		err := yaml.Unmarshal(file.Data, &address)
+		if err != nil {
+			return ipAddresses, errors.Wrapf(err, "unmarshal ip address %d", i)
```

Comment:
change to: "unable to unmarshal ipaddress %d"

Reply:
I was using same error message from the lines prior to this where the same activities are being done.  For consistency, should I also update the other error messages to add "unable to "?

Reply:
i dont think that's necessary, was just thinking it might clarify the error.